docs/refaster.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Refaster templates
+layout: documentation
+---
+
+In addition to [patching] your code using the checks built-in to Error Prone, we've developed a mechanism to refactor your code using before-and-after templates (we call them Refaster templates). Once you write these templates, you compile them into .refaster files, then use the Error Prone compiler to refactor your code according to those rules.
+
+## Building Refaster Templates
+
+Writing refaster rules is best done by example:
```

Comment:
This sentence is a little off, should it be something like "Explaining how to write refaster rules is best done by example"?